src/utils/object.ts

```diff
@@ -24,11 +25,11 @@ export const transformKeys = (
     transformer: Transformer,
     ignoreKeys: string[] = []
 ): any => {
-    const transformArray = (arr: unknown[], transformer: Transformer): any =>
+    const transformArray = (arr: unknown[]): any =>
         arr.map((item) => (isObject(item) ? transformKeys(item, transformer, ignoreKeys) : item));
 
     if (Array.isArray(obj)) {
-        return transformArray(obj, transformer);
+        return transformArray(obj);
```

Comment:
Yeah... This is not really related but there was a shadow variable... Better fix that now 🙂